mw-webapp/src/component/image/Image.stories.ts

```diff
@@ -0,0 +1,14 @@
+import type {StoryObj} from "@storybook/react";
+import {Image} from "src/component/image/Image";
+
+const meta = {
+  tile: "Image",
+  component: Image,
+  parameters: {layout: "centered"},
+  tags: ["autodocs"],
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {args: {src: "https://picsum.photos/200/300", alt: "Image"}};
```

Comment:
wow) can you guarantee that this image will be exist tomorrow? What happened with preview in story book if image disappeared? it's gonna surprise the developers a little bit.

Reply:
why alt - image? does it semantically correct?

Reply:
1. I can't guarantee that the source of the image will still exist. But what can I use as a preview instead?
2. Do you want me to write more descriptive text like "Default storybook picture"?

Reply:
1. You can download any picture and use it for story
2. Yes need to add description but it depends on which picture.

Reply:
Should I place an image in the same folder of the component, or should I create an 'assets' folder?

Reply:
Create folder for images